man/git-extras.html

```diff
@@ -1,194 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Did `ronn` fail to generate `git-extras.html` on your side?

Reply:
since i'm at work right now, i cant check that but i dont remember seeing an error after execution in WSL.

i just had to assume that this is desired result b/c i dont know what the end goal has to look like.

Reply:
i checked and i didnt see any errors on my WSL console (was still open)
but i reran the commands again and pushed that change, i guess that looks better now.